src/main/java/com/khutircraftubackend/category/CategoryMapper.java

```diff
@@ -27,5 +30,9 @@ public interface CategoryMapper {
     @Mapping(target = "parentCategory.id", source = "request.parentCategoryId", nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
     void updateCategoryEntity(@MappingTarget CategoryEntity category, CategoryUpdateRequest request);
 
-
+ default List<CategoryResponse> toCategoryResponseList(List<CategoryEntity> categoryEntityList) {
+     return categoryEntityList.stream()
+             .map(this::toCategoryResponse)
+             .collect(Collectors.toList());
+ }
```

Comment:
i'd suggest using something more generic, rather than list. 
what if for some reason we would like to have a Set instead (just because Set is a collection of unique objects)?
so Collection would probably fit better here.

method name can be the same: `toCategoryResponse()` - we'll use OOP's nice feature - method overloading in that case.

method's signature: 
```java
default Collection<CategoryResponse> toCategoryResponse(Collection<CategoryEntity> categoryEntities) { ... }
// gets argument: Collection<CategoryEntity> categoryEntities
// as plural for "categoryEntity"
```

oh, and there are some issues with indents in this file as well.